### PR TITLE
Roles configs

### PR DESCRIPTION
**Pull request by rajivnathan**
Adds roles for the new `toolchainconfigs` and `memberoperatorconfigs` CRDs.

Related PR: https://github.com/codeready-toolchain/api/pull/236

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Comment by rajivnathan** [comment]
@MatousJobanek I replied to the original comment https://github.com/codeready-toolchain/api/pull/236#discussion_r635054907 The plan is to delete the host operator CRD and controller once we're using the toolchainconfig CRD and controller so we can delete the hostoperatorconfig.go at that time as well. But as always, open to any better suggestions.

File: scripts/add-cluster.sh [hunk 1]
```diff
@@ -67,6 +67,7 @@ rules:
   - "registrationservices"
   - "templateupdaterequests"
   - "tiertemplates"
+  - "toolchainconfigs"
```
Comment (MatousJobanek) [inline scripts/add-cluster.sh:70]: Please ping me when you merge the PR, so I can update the Roles

File: pkg/test/toolchainconfig.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+func NewToolchainConfig(options ...HostConfigOption) *toolchainv1alpha1.ToolchainConfig {
+	toolchainConfig := &toolchainv1alpha1.ToolchainConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: HostOperatorNs,
+			Name:      "config",
+		},
+	}
+	for _, option := range options {
+		// TODO: pass toolchainConfig directly as param once HostOperatorConfig is removed, see https://issues.redhat.com/browse/CRT-1120
```
Comment (MatousJobanek) [inline pkg/test/toolchainconfig.go:118]: I'm not sure what is meant by this comment, but I'll leave it up to you :wink:

**Review by MatousJobanek — APPROVED** [review]
:+1:

**Pull request merged**